src/components/coin-page/CoinSummary/CoinSummary.js

```diff
@@ -56,8 +56,29 @@ import {
 } from "./CoinSummary.styles";
 
 const CoinSummary = (props) => {
-
-  const activeCurrency = useSelector(getActiveCurrency)
+  const { name: currencyName, symbol: currencySymbol } =
+    useSelector(getActiveCurrency);
+  const {
+    image: coinImage,
+    name: coinName,
+    symbol: coinSymbol,
+    links: coinLinks,
+    market_data: coinMarketData,
+  } = props.coin;
+  const {
+    current_price: currentPrice,
+    price_change_percentage_24h: priceChangePercentage24hr,
+    price_change_24h_in_currency: priceChange24hrInCurrency,
+    ath,
+    atl,
+    ath_date: athDate,
+    atl_date: atlDate,
+    market_cap: marketCap,
+    fully_diluted_valuation: fullyDilutedValuation,
+    total_volume: totalVolume,
+    circulating_supply: circulatingSupply,
+    max_supply: maxSupply,
+  } = coinMarketData;
```

Comment:
you can make a function that converts all object keys from snake case to camel case... and you can use that everywhere...